packages/nx/src/command-line/show/project.ts

```diff
@@ -20,7 +20,7 @@ export async function showProjectHandler(
         view: 'project-details',
         focus: node.name,
         watch: true,
-        open: true,
```

Comment:
We should add this in more places right? I think `graph.ts` at least also needs to take it into account... 

For `nx build app --graph`  I don't think we should have this arg... it'll conflict with many other tools.

Reply:
`nx graph` already accepts `--open`

Reply:
Brilliant